wrapanapi/systems/openstack.py

```diff
@@ -173,6 +173,17 @@ def all_ips(self):
         # raw.networks is dict of network: [ip, ip] key:value pairs
         return [ip for nets in self.raw.networks.values() for ip in nets]
 
+    @property
+    def floating_ip(self):
+        return self.get_ip_by_type('floating')
+
+    @property
+    def ip(self):
+        floating_ip = self.floating_ip
+        if floating_ip:
+            return floating_ip
+        return self.get_ip_by_type('fixed')
```

Comment:
This is a change to the behavior of this property - I think it should still return `None` if there is no floating IP found. Right now the caller expects this `ip` property to return a 'reachable' address, not an internal/fixed address, and `None` if that is not found.

With this change, the caller will now not know whether the returned address is the floating address.

I would suggest continuing to return `None` here if a floating IP isn't found, and the caller can then resort to directly calling `get_ip_by_type` directly if they would like to proceed with a fixed address.

Reply:
removed and let default ip property to be equal to floating ip, as some tests are expecting this value to be floating ip.